README.md

```diff
@@ -125,42 +125,24 @@ NOTE: The url of the jupyter server will be displayed in the docker-compose outp
 (default: `http://127.0.0.1:8888/?token=SOME_JUPYTER_TOKEN`) (you can customize jupyter port and host using [env](#env-variables) variables)
 
 ### Training
-Currently GPU support is available only in the experimental jupyter notebook `neural_cf` that contains whole ML cycle:
-- Datasets creation, preprocessing and inspections
-- Models creation (Autoencoders, NCF)
-- Trainings
-- Evaluation
-- Inferention
 
-GPU support can be enabled using environmental variable TRAINING_DEVICE (look into [ENV variables](#env-variables) section) , but for now it doesn't work in the dev/test/prod environments due to the fact that celery uses `fork` rather than `spawn` multiprocessing method - it is incompatibile with `CUDA`.
+Recommender system can use one of two recommendation engines implemented as agents:
+- `Pre Agent` - that implements [Neural Collaborative Filtering](https://arxiv.org/abs/1708.05031) algorithm.
+- `RL Agent` - that implements [Deep Deterministic Policy Gradient](https://arxiv.org/abs/1509.02971) algorithm.
 
-For now the only trainable recommendation agent is the Pre-Agent.
+To choose prefered agent, use `AGENT_VERSION` env variable (look into [ENV variables](#env-variables) section). After change of this variable in the .env file no reloading is needed - system will use appropriate agent dynamically (if it is trained).
 
-It can be trained via training endpoint (used in production) or via training tasks.
+For now the only trainable recommendation agent is the Pre Agent. RL Agent training will be available soon.
 
-Before training, the development database should be populated. It can be accomplished in two ways:
- - posting marketplace database dump to the `database_dumps` endpoint
- - loading it manually using `load_mp_dump` function
+The simplest way to train chosen agent is using `./bin/rails recommender:update` task on the Marketplace side to send request to the `/update` endpoint of the Recommender System. It automatically sends most recent training data, preprocesses and uses them to train needed models.
```

Comment:
[..] to train a chosen agent, is using the [command] on the Marketplace side. It sends a request [...]. It automatically uses the most recent [...] preprocesses and uses IT to train [...]